resources/migrations/000_migrations.yaml

```diff
@@ -5464,6 +5707,17 @@ databaseChangeLog:
                 WHERE dbcl.id = '84'
                   AND metric.updated_at < dbcl.dateexecuted
               )
+        - sql:
+            dbms: mysql,mariadb
+            sql: >-
+              UPDATE metric
+              SET archived = NOT archived
+              WHERE EXISTS (
+                SELECT *
+                FROM `DATABASECHANGELOG` dbcl
+                WHERE dbcl.id = '84'
+                  AND metric.updated_at < dbcl.dateexecuted
+              )
```

Comment:
Unreleated, but this migration was actually a little busted -- it checked whether `databasechangelog` exists before running. `databasechangelog` should ALWAYS exist, but the precondition failed for MySQL because it's named `DATABASECHANGELOG` (all caps) for mystery reasons. So effectively this migration would never run for MySQL or MariaDB.

I reworked this migration so it did the correct thing -- use slightly different SQL for MySQL/MariaDB and remove the precondition.